doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-coverage-report-dmn-based-proc.adoc

```diff
@@ -1,13 +1,23 @@
 [id='test-scenarios-coverage-report-dmn-based-proc']
 = Generating coverage reports for DMN-based test scenarios
 
-Follow the procedure below to generate a coverage report for DMN-based test scenarios.
+In DMN-based test scenarios, the *Coverage Report* tab contains the detailed information about the following:
+
+* Number of available decisions
+* Number of executed decisions
+* Percentage of executed decisions,
+* Percentage of executed decisions represented as a pie chart
+* Number of times each decision has executed
+* Decisions that are executed for each defined test scenario
+
+Follow the procedure to generate a coverage report for DMN-based test scenarios:
 
 .Prerequisites
-* The DMN-based test scenario template has been created for the selected test scenario. For more information about creating DMN-based test scenarios, see xref:test-designer-create-test-template-dmn-based-proc[].
-* The individual test scenarios have been defined. For more information about defining a test scenario, see xref:test-designer-test-scenario-definition-proc[].
+* The DMN-based test scenario template is created for the selected test scenario. For more information about creating DMN-based test scenarios, see xref:test-designer-create-test-template-dmn-based-proc[].
+* The individual test scenarios are defined. For more information about defining a test scenario, see xref:test-designer-test-scenario-definition-proc[].
 
 .Procedure
 . Open the DMN-based test scenarios in the test scenario designer.
 . Run the defined test scenarios.
 . Click *Coverage Report* on the right of the test scenario designer to display the test coverage statistics.
+. Optional: To download the test scenario coverage report, Click *Download report*.
```

Comment:
I have a suggestion to keep the optional the same as at 3.4.
Like "Optionally, to download the test scenario coverage report, Click *Download report*.